include/pmacc/eventSystem/tasks/TaskSend.hpp

```diff
@@ -97,14 +121,14 @@ namespace pmacc
 
         void event(id_t, EventType type, IEventData*)
         {
-            if (type == COPYDEVICE2HOST)
+            if(type == COPYDEVICE2HOST || type == COPYDEVICE2DEVICE)
             {
                 state = DeviceToHostFinished;
                 executeIntern();
 
             }
```

Comment:
Not saying this is an error, but `state = DeviceToHostFinished;` looks unexpected given the modified condition,